helios-client/src/main/java/com/spotify/helios/client/HeliosClient.java

```diff
@@ -609,6 +610,9 @@ public static Builder newBuilder() {
     private boolean sslHostnameVerification = true;
     private boolean googleCredentialsEnabled = true;
     private AccessToken googleAccessToken;
+    private List<String> googleAccessTokenScopes = ImmutableList.of(
```

Comment:
What about moving these to `GoogleCredentialsAccessTokenProvider.DEFAULT_SCOPES`?